<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>mainMenuView</class>
 <widget class="QWidget" name="mainMenuView">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1200</width>
    <height>850</height>
   </rect>
  </property>
  <property name="styleSheet">
   <string notr="true">background: #8A8557;</string>
  </property>
  <widget class="QPushButton" name="newGameButton">
   <property name="geometry">
    <rect>
     <x>470</x>
     <y>280</y>
     <width>261</width>
     <height>31</height>
    </rect>
   </property>
   <property name="sizePolicy">
    <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
     <horstretch>0</horstretch>
     <verstretch>0</verstretch>
    </sizepolicy>
   </property>
   <property name="font">
    <font>
     <pointsize>12</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: #111111;
color: rgb(171, 171, 0);
text-fill: #8A8557;
border-color: #47452E;
pref-width:200;
pref-height: 30;
</string>
   </property>
   <property name="text">
    <string>Neues Spiel</string>
   </property>
   <property name="autoDefault">
    <bool>false</bool>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>480</x>
     <y>210</y>
     <width>241</width>
     <height>41</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>48</pointsize>
     <bold>false</bold>
    </font>
   </property>
   <property name="text">
    <string>ETANKS</string>
   </property>
  </widget>
  <widget class="QPushButton" name="showProfilButton">
   <property name="geometry">
    <rect>
     <x>470</x>
     <y>330</y>
     <width>261</width>
     <height>31</height>
    </rect>
   </property>
   <property name="sizePolicy">
    <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
     <horstretch>0</horstretch>
     <verstretch>0</verstretch>
    </sizepolicy>
   </property>
   <property name="font">
    <font>
     <pointsize>12</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: #47452E;
     text-fill:#111111;
     border-color: #111111;
     border-width: 2;
     pref-width:200;
     pref-height: 30;
</string>
   </property>
   <property name="text">
    <string>Profil</string>
   </property>
   <property name="autoDefault">
    <bool>false</bool>
   </property>
  </widget>
  <widget class="QPushButton" name="showStatisticButton">
   <property name="geometry">
    <rect>
     <x>470</x>
     <y>380</y>
     <width>261</width>
     <height>31</height>
    </rect>
   </property>
   <property name="sizePolicy">
    <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
     <horstretch>0</horstretch>
     <verstretch>0</verstretch>
    </sizepolicy>
   </property>
   <property name="font">
    <font>
     <pointsize>12</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: #47452E;
     text-fill:#111111;
     border-color: #111111;
     border-width: 2;
     pref-width:200;
     pref-height: 30;
</string>
   </property>
   <property name="text">
    <string>Statisik</string>
   </property>
   <property name="autoDefault">
    <bool>false</bool>
   </property>
  </widget>
  <widget class="QPushButton" name="showSettingsButton">
   <property name="geometry">
    <rect>
     <x>470</x>
     <y>430</y>
     <width>261</width>
     <height>31</height>
    </rect>
   </property>
   <property name="sizePolicy">
    <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
     <horstretch>0</horstretch>
     <verstretch>0</verstretch>
    </sizepolicy>
   </property>
   <property name="font">
    <font>
     <pointsize>12</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: #47452E;
     text-fill:#111111;
     border-color: #111111;
     border-width: 2;
     pref-width:200;
     pref-height: 30;
</string>
   </property>
   <property name="text">
    <string>Einstellungen</string>
   </property>
   <property name="autoDefault">
    <bool>false</bool>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
